Mathlib/Analysis/Convex/Extreme.lean

```diff
@@ -236,6 +232,23 @@ theorem extremePoints_pi (s : ∀ i, Set (π i)) :
 
 end OrderedSemiring
 
+section OrderedRing
+variable [OrderedRing 𝕜] [AddCommGroup E] [Module 𝕜 E] [AddCommGroup F] [Module 𝕜 F]
+
+lemma image_extremePoints (f : E ≃ₗ[𝕜] F) (s : Set E) :
```

Comment:
Should this live in a namespace?

Reply:
`extremePoints` is not in a namespace

Reply:
I meant `LinearEquiv.image_extremePoints`. Or use a typeclass assumption on the type of `f` (do we have `LinearEquivClass`?). BTW, this is true for affine equivalences as well.

Reply:
Switched to using `LinearEquivClass`. You can see why I originally didn't want to do that.

Reply:
We have functions "from class to hom" for some other homs. I think that we should either add it here, or generalize relevant parts to `*Class`. I can have a look in ~1h.

Reply:
I golfed. I think that we should add a definition `SemilinearMapClass.toLinearMap` but this may wait.

bors d+